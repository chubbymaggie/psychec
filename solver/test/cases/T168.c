void f()
{
    char *c;
    const char* n;
    quote(c);
    quote(n);
    char *group_name = c;
}
