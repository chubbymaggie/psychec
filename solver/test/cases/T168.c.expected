/* Forward declarations */

/* Type definitions */

/* Variables and functions */
 int /*<<< orphan*/  quote (char const*) ; 

